src/fides/api/models/privacy_experience.py

```diff
@@ -612,14 +621,14 @@ def upsert_privacy_experiences_after_config_update(
     Assumes that components on the ExperienceConfig do not change after they're updated.
     """
     current_regions: List[PrivacyNoticeRegion] = experience_config.regions
-    removed_regions: List[
-        PrivacyNoticeRegion
-    ] = [  # Regions that were not in the request, but currently attached to the Config
-        PrivacyNoticeRegion(reg)
-        for reg in {reg.value for reg in current_regions}.difference(
-            {reg.value for reg in regions}
-        )
-    ]
+    removed_regions: List[PrivacyNoticeRegion] = (
```

Comment:
hmmm i guess my formatter changed this 🤔 

i can remove the change if we want.